nwbwidgets/base.py

```diff
@@ -182,7 +182,7 @@ def lazy_show_over_data(list_, func_, labels=None, style: GroupingWidget = widge
     children = [vis2widget(func_(list_[0]))] + [widgets.HTML("Rendering...") for _ in range(len(list_) - 1)]
     out = style(children=children)
     if labels is not None:
-        [out.set_title(i, label) for i, label in enumerate(labels)]
+        [out.set_title(i, str(label)) for i, label in enumerate(labels)]
```

Comment:
@bendichter This fixed it

I guess one of the labels in the test cases was actually an integer, and at some point they stopped magically casting it as a string during `set_title`

We should *probably* do it elsewhere we set titles as well just to be sure, but that can be separate PR